examples/RemoveObjects.java

```diff
@@ -48,6 +48,7 @@ public static void main(String[] args)
       objectNames.add("my-objectname3");
 
       // Remove object all objects 'objectNames' list from 'my-bucketname'.
+      // Lazy evaluation. Need to traverse over the returned Iterable to do the real delete jobs.
```

Comment:
You could add comment like `Its required to traverse over the returned Iterable for lazy evaluation`